cpp/src/c_api/generic_cascaded_dispatch.hpp

```diff
@@ -132,9 +132,10 @@ constexpr decltype(auto) edge_type_type_dispatcher(cugraph_data_type_id_t edge_t
     }
 
     default: {
-      std::stringstream ss;
-      ss << "ERROR: Unknown type enum:" << static_cast<int>(edge_type_type);
-      throw std::runtime_error(ss.str());
+      // std::stringstream ss;
```

Comment:
maybe put a comment 
// needs c++23 to have std::stringstream inside a constexpr function